<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgt_marker_table</class>
 <widget class="QWidget" name="wgt_marker_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>280</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="wgt_marker_editor" name="wgt_marker_edit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtButtons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>214</width>
       <height>16000000</height>
      </size>
     </property>
     <widget class="QPushButton" name="cmdAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdRemove">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/minus</normaloff>:/icons/minus</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdRemoveAll">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdMerge">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdDirectionFind">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>184</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Direction Find</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdLoadMarkers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>184</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Markers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdSaveMarkers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>184</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Markers</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblMarkers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Courier 10 Pitch</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wgt_marker_editor</class>
   <extends>QWidget</extends>
   <header>wgt_marker_editor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
